packages/ui/.storybook/main.js

```diff
@@ -25,5 +25,8 @@ const config = {
   docs: {
     autodocs: 'tag',
   },
+  typescript: {
+    reactDocgen: 'react-docgen-typescript',
```

Comment:
Auto-generates Storybook docs based on TypeScript typings of props, e.g.:
<img width="1482" alt="image" src="https://github.com/penumbra-zone/web/assets/1121544/3508cf4d-2cb3-4c59-8619-f98580191cc9">